### PR TITLE
Testsuite: Fix cucumber Ansible tests

### PR DESCRIPTION
**Pull request by meaksh**
## What does this PR change?

This PR fixes the current failures in the Ansible cucumber scenarios after latest changes.

## GUI diff

No difference.

- [x] **DONE**

## Documentation
- No documentation needed: **testsuite**

- [x] **DONE**

## Test coverage
- Cucumber tests were added

- [x] **DONE**

## Links

- [x] **DONE**

## Changelogs

If you don't need a changelog check, please mark this checkbox:

- [x] No changelog needed

If you uncheck the checkbox after the PR is created, you will need to re-run `changelog_test` (see below)

## Re-run a test

If you need to re-run a test, please mark the related checkbox, it will be unchecked automatically once it has re-run:

- [ ] Re-run test "changelog_test"
- [ ] Re-run test "backend_unittests_pgsql"
- [ ] Re-run test "java_pgsql_tests"
- [ ] Re-run test "schema_migration_test_oracle"
- [ ] Re-run test "schema_migration_test_pgsql"
- [ ] Re-run test "susemanager_unittests"
- [ ] Re-run test "javascript_lint"
- [ ] Re-run test "spacecmd_unittests"

### CODE REVIEW

File: testsuite/features/secondary/min_ansible_control_node.feature [hunk 1]
```diff
@@ -5,7 +5,14 @@
 Feature: Operate an Ansible control node in a normal minion
 
   Scenario: Pre-requisite: Deploy test playbooks and inventory file
-    Given I deploy testing playbooks and inventory files to "sle_minion"
+    Given I am on the Systems overview page of this "sle_minion"
+    Then I deploy testing playbooks and inventory files to "sle_minion"
```
Comment (srbarrios) [inline testsuite/features/secondary/min_ansible_control_node.feature:9]: When

File: testsuite/features/secondary/min_ansible_control_node.feature [hunk 2]
```diff
@@ -5,7 +5,14 @@
...
+
+  Scenario: Pre-requisite: Enable client tools repositories
+    Given I am on the Systems overview page of this "sle_minion"
+    Then I enable SUSE Manager tools repositories on "sle_minion"
```
Comment (srbarrios) [inline testsuite/features/secondary/min_ansible_control_node.feature:13]: When

File: testsuite/features/secondary/min_ansible_control_node.feature [hunk 3]
```diff
@@ -74,3 +81,9 @@ Feature: Operate an Ansible control node in a normal minion
     Then I should see a "System properties changed" text
     And I remove package "orion-dummy" from this "sle_minion" without error control
     And I remove "/tmp/file.txt" from "sle_minion"
+
+  Scenario: Cleanup: Disable client tools channel
+    Given I am on the Systems overview page of this "sle_minion"
+    Then I disable SUSE Manager tools repositories on "sle_minion"
```
Comment (srbarrios) [inline testsuite/features/secondary/min_ansible_control_node.feature:87]: When

File: testsuite/features/secondary/minssh_ansible_control_node.feature [hunk 4]
```diff
@@ -7,7 +7,14 @@
 Feature: Operate an Ansible control node in SSH minion
 
   Scenario: Pre-requisite: Deploy test playbooks and inventory file
-    Given I deploy testing playbooks and inventory files to "ssh_minion"
+    Given I am on the Systems overview page of this "ssh_minion"
+    Then I deploy testing playbooks and inventory files to "ssh_minion"
```
Comment (srbarrios) [inline testsuite/features/secondary/minssh_ansible_control_node.feature:11]: When

File: testsuite/features/secondary/minssh_ansible_control_node.feature [hunk 5]
```diff
@@ -76,3 +83,9 @@ Feature: Operate an Ansible control node in SSH minion
     Then I should see a "System properties changed" text
     And I remove package "orion-dummy" from this "ssh_minion" without error control
     And I remove "/tmp/file.txt" from "ssh_minion"
+
+  Scenario: Cleanup: Disable client tools channel
+    Given I am on the Systems overview page of this "ssh_minion"
+    Then I disable SUSE Manager tools repositories on "ssh_minion"
```
Comment (srbarrios) [inline testsuite/features/secondary/minssh_ansible_control_node.feature:89]: When

**Review by srbarrios — APPROVED** [review]

**Comment by meaksh** [comment]
@srbarrios I've fixed the "When/Then" tags, I'll wait for tests to pass and will merge this. Thanks!

**Pull request merged**